guides/src/components/MarkdownPageFooter.js

```diff
@@ -0,0 +1,29 @@
+// --- Dependencies
+import * as React from 'react'
+import PropTypes from 'prop-types'
+
+/**
+ * Component
+ */
+
+export default function MarkdownPageFooter({ group, isIndex, section, title }) {
+
+  const baseUrl = 'https://github.com/spark-solutions/spree/blob/master/guides/src/content'
+  const edge = group
+    .find(el => el.section === section).edges
+    .find(el => el.node.frontmatter.title === title)
+  const pathname = edge ? edge.node.fields.slug : window.location.pathname
```

Comment:
@AleksandraKaminska if there's no `edge` we shouldn't render this button at all, probably we won't have the source `.md` file in place (some pages are in JS)